mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -11800,17 +11792,6 @@ resource "google_container_cluster" "storage_pools_with_node_config" {
 `, cluster, location, networkName, subnetworkName, storagePoolResourceName)
 }
 
-func extractSPName(url string) (string, error) {
-    re := regexp.MustCompile(`https://www\.googleapis\.com/compute/beta/(projects/[^"]+)`)
```

Comment:
for sure, updating this to
`/(beta|v1)` worked, but reading through the past issues / comments, thinking something like what's in here is closer to what you'd want?

Reply:
Yea I think the tpgresource.GetRelativePath() is a good idea